lib/teaspoon/mocha/integration.rb

```diff
@@ -0,0 +1,36 @@
+module Teaspoon
+  module Mocha
+    JAVASCRIPT_VERSIONS = {
+      "1.10.0" => ["mocha/1.10.0", "teaspoon-mocha"],
+      "1.17.1" => ["mocha/1.17.1", "teaspoon-mocha"],
+    }
+
+    SUPPORT_LIBRARIES = [
+      'chai',
+      'chai-1.10.0',
+      'chai-jq-0.0.7',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.